resources/web/wwi/nodes/WbGeometry.js

```diff
@@ -126,8 +126,10 @@ export default class WbGeometry extends WbBaseNode {
   _deleteWrenRenderable() {
     if (typeof this._wrenRenderable !== 'undefined') {
       // Delete picking material
-      _wr_material_delete(Module.ccall('wr_renderable_get_material', 'number', ['number', 'string'], [this._wrenRenderable, 'picking']));
-      _wr_material_delete(Module.ccall('wr_renderable_get_material', 'number', ['number', 'string'], [this._wrenRenderable, 'depth']));
+      _wr_material_delete(Module.ccall('wr_renderable_get_material', 'number', ['number', 'string'], [this._wrenRenderable,
+        'picking']));
+      _wr_material_delete(Module.ccall('wr_renderable_get_material', 'number', ['number', 'string'], [this._wrenRenderable,
+        'depth']));
```

Comment:
The best practice would be to split the line at a higher level
```suggestion
      _wr_material_delete(Module.ccall('wr_renderable_get_material', 'number', ['number', 'string'],
        [this._wrenRenderable, 'picking']));
      _wr_material_delete(Module.ccall('wr_renderable_get_material', 'number', ['number', 'string'],
        [this._wrenRenderable, 'depth']));
```